manager.java

```diff
@@ -1,70 +1,51 @@
 package cinema;
 import java.util.Scanner;
-import java.lang.Math;
 
 public class Cinema {
```

Comment:
This class is named Cinema and is in a file named manager.java

Reply:
corrected in recent commit